src/h5web/visualizations/heatmap/HeatmapVis.tsx

```diff
@@ -30,24 +34,74 @@ function HeatmapVis(props: Props): JSX.Element {
     keepAspectRatio = true,
     showGrid = false,
     showLoader = true,
+    abscissas,
+    ordinates,
```

Comment:
You can move calls to `range` here like in `LineVis`.

Reply:
Can I really ? I need `rows` and `cols` for that, that are returned from `getDims`

Reply:
Hmm, you can have a second destructuring of `props` after the call to `getDims`. I think it's still nicer than to have an extra `abscissaValues` constant.